plugins/html-elements-plugin/src/main/java/ru/sbtqa/tag/pagefactory/utils/HtmlReflectionUtils.java

```diff
@@ -413,56 +412,56 @@ public static void executeMethodByTitleInBlock(Page page, String blockPath, Stri
      * @throws ru.sbtqa.tag.pagefactory.exceptions.PageException TODO
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)